src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -241,9 +241,12 @@ public function rsvp_process() {
 			wp_send_json_error( $response );
 		}
 
-		$has_tickets = false;
-		$post_id     = get_the_id();
-		$ticket      = tribe( 'tickets.rsvp' )->get_ticket( $post_id, $ticket_id );
+        /** @var Tribe__Tickets__RSVP $rsvp */
+        $rsvp           = tribe( 'tickets.rsvp' );
```

Comment:
Indentation is weird here?

Reply:
ok, I'll take a look. Seems like there is a difference with my use of spaces as tabs.